geoportailv3/static/js/layerinfo/popup.html

```diff
@@ -0,0 +1,5 @@
+<h4 class="popover-title ngeo-popup-title">
+  <span>{{title | translate}}</span>
+  <button type="button" class="close" ng-click="open = false"> &times;</button>
+</h4>
+<div class="popover-content" ng-bind-html="content"></div>
```

Comment:
there are two sides to this.
it will be presented as a table with titles in the first td and content in the second td.

The first column has to be translated, the content comes directly translated, but the language has to included in the call

eg: http://shop.geoportail.lu/Portail/inspire/webservices/getMD.jsp?uid=ae1bc422-5751-4efa-92af-c24287609dbb&lang=fr&_dc=1422894522958&cb=stcCallback1002